### DISCUSSION TITLE
Google Wallet Not showing in Configuration

### DISCUSSION THREAD

**Discussion by wesleypearson** [alfio-event/alf.io#1242] [Q&A] [answered]
Hello! Using the latest build and can not see where to configure the Google Wallet details under Mobile Wallet Settings

**Comment by cbellone** [comment, marked as answer]
Hi! Support for Google Wallet was merged into the main branches, but not yet released.
If you want to use it, you have to build and deploy from the source code. Otherwise, you must wait for the next release (no ETA yet, but should be relatively soon)
